### DISCUSSION TITLE
Game directory is corrupted

### DISCUSSION THREAD

**Discussion by purplelunamine** [Monika-After-Story/MonikaModDev#9255] [QA - tech support + general]
I can't open the game, whenever I do, it throws an error saying that there's a corrupted directory. Because of that, I decided to re-download the game and replace all the existing files with the ones from the .zip. However, the error is still there. [Here's the traceback.](https://github.com/Monika-After-Story/MonikaModDev/files/8935762/traceback.txt)

How can I fix this, and why is it happening?

**Comment by Booplicate** [comment]
We don't have that package, it's probably some other mod.
